### DISCUSSION TITLE
xml file for the SLC does not exist, you need to have these

### DISCUSSION THREAD

**Discussion by himanshuchaube** [isce-framework/isce2#628] [Q&A]
After successfully running all run_files it got stuck with "xml file for the SLC does not exist, you need to have these" error when I am giving "make_single_reference_stack_isce input_file."

![Screenshot from 2022-11-22 19-00-34](https://user-images.githubusercontent.com/102573714/203326586-6df594d2-fb99-4833-8f7c-c5c89a3c2e8d.png)

Please suggest how to rectify this error?

**Comment by bqh6741** [comment]
hello，I have the same question. Could you pliease tell me how to solute it?  
![图片](https://github.com/isce-framework/isce2/assets/120152998/98ec9517-a252-4c7a-85a3-66b9fdad2ab7)
 I thinkI have no missing files.

**Reply by himanshuchaube** [reply]
I did not get its solution but, with some other way, I could able to solve it. Can you drop me your contact details so we can discuss it in detail?

**Comment by bqh6741** [comment]
Thank you very much! I ' m trying to rerun the 13 steps in runfiles to see if there ' s a bug.

**Reply by himanshuchaube** [reply]
are you processing using matlab or mintpy?

**Comment by bqh6741** [comment]
Hi，stamps matlab

**Reply by himanshuchaube** [reply]
I have forgotten some parts of the processing as I did it during training, can you drop me your contact details so that I can also restart the process and could able to do it together?

**Comment by bqh6741** [comment]
6901266465@qq.com This is my mailbox and my qq number

**Reply by ANNYIN123** [reply]
hello! I meet the same question, have you solve it?  2991519556@qq.com .This is my email ,can we have some communication?

**Reply by donghumeilang** [reply]
hello! I meet the same question, have you solve it?

**Comment by 000000ta** [comment]
你们看看input file里面的路径有没有设置对？我路径设置错，出现了这个问题。

**Comment by yangs0001** [comment]
https://groups.google.com/g/mainsar/c/-AH-NLskKxw
